generate-routes.js

```diff
@@ -8,22 +8,33 @@ import {
 import { openapi } from '@seamapi/types/connect'
 import { deleteAsync } from 'del'
 
-const libNameParts = ['lib', 'seam']
-const libPrefix = posix.join(...libNameParts)
-const libVersionPath = posix.join(libPrefix, 'version.rb')
-
 const rootPath = dirname(fileURLToPath(import.meta.url))
+const libName = 'lib'
+const libPath = posix.join(rootPath, libName)
+const seamName = 'seam'
+const seamPath = posix.join(libPath, seamName)
+const libVersionPath = posix.join(seamPath, 'version.rb')
+const libVersionRelativePath = posix.join(libName, seamName, 'version.rb')
+
+const pathsToDelete = [
+  `${libPath}/**/*`,
+  `!${libPath}/seam/`,
+  `!${libVersionPath}`,
```

Comment:
`[`${libPrefix}/**`, `!${libVersionPath}`]` wasn't working for me, it was deleting everything inside `lib` directory including `lib/seam/version.rb`. Adding `!${libPath}/seam/` did the trick